tests/integration/authentication/test_401_errors.py

```diff
@@ -0,0 +1,25 @@
+import requests
+import pytest
+from lib import Assertions, Generators
+
+TOKENS = [None, "Bearer xyzcba", "Bearer", "junk"]
+METHODS = ["get", "post", "put", "patch", "delete", "head", "options"]
+CORRELATION_IDS = [None, "76491414-d0cf-4655-ae20-a4d1368472f3"]
+
+
+@pytest.mark.inttest
+@pytest.mark.parametrize('invalid_token', TOKENS)
+@pytest.mark.parametrize("correlation_id", CORRELATION_IDS)
+@pytest.mark.parametrize("method", METHODS)
+def test_401_invalid(nhsd_apim_proxy_url, invalid_token, correlation_id, method):
+    resp = getattr(requests, method)(f"{nhsd_apim_proxy_url}", headers={

```

Comment:
The `nhsd_apim_proxy_url` here should be swapped over to use `https://int.api.service.nhs.uk/comms` - this value should be put into our constants and imported into the files.

Our int tests should run like an API consumers are going to run.

Reply:
replaced nhsd_apim_proxy with constant INT_URL and just call that for the test